rdmo/management/management/commands/merge_attributes.py

```diff
@@ -0,0 +1,228 @@
+    qs = get_queryset(model, lookup_field, field_introspection, attribute=source)
+    replaced_attributes = []
+    for instance in qs:
+        instance = replace_attribute_on_instance(instance, lookup_field, source, target, field_introspection,
+                                                 replacement_function)
+        if save:
+            instance.save()
+        replaced_attributes.append({
+            'model_name': model._meta.verbose_name_raw,
+            'instance': instance,
+            'source': source,
+            'target': target,
+            'saved': save,
+            'source_exists': get_queryset(model, lookup_field, field_introspection, attribute=source).exists()
+        })
+    return replaced_attributes
+
+
+def get_valid_attribute(attribute, message_name=''):
+    if isinstance(attribute, str):
+        attribute_uri = attribute
+        try:
+            attribute = Attribute.objects.get(uri=attribute_uri)
+        except Attribute.DoesNotExist as e:
+            raise ValueError(f"{message_name} attribute {attribute_uri} does not exist.") from e
+        except Attribute.MultipleObjectsReturned as e:
+            raise ValueError(f"{message_name} attribute {attribute_uri} returns multiple objects.") from e
+    elif not isinstance(attribute, Attribute):
+        raise ValueError(f"{message_name} argument should be of type Attribute.")
+
+    return attribute
+
+def make_elements_message(results):
+    element_counts = ", ".join([f"{k.capitalize()}({len(v)})" for k, v in results.items() if v])
+    if not element_counts or not any(results.values()):
+        return "No elements affected."
+    return f"Affected elements: {element_counts}"
+
+
+def make_affected_projects_message(results):
+    value_results = results.get(Value._meta.verbose_name_raw, [])
+    if not value_results:
+        return "No projects affected."
+    msg = "Affected Projects:"
+    for project in {i['instance'].project for i in value_results}:
+        msg += f"\n\tproject={project} (id={project.id})"
+    return msg
+
+
+def make_affected_instances_message(results):
+    if not results:
+        return ""
+    msg = ""
+    for k, merger_results in results.items():
+        if not merger_results:
+            continue
+        msg += f"Merger results for model {k.capitalize()} ({len(merger_results)})"
+        msg += '\n'
+        for result in merger_results:
+            msg += dedent(f'''\
+                Model={result['model_name']}
+                instance={result['instance']}
+                source={result['source'].uri}
+                source_exists={result['source_exists']}
+                target={result['target'].uri}
+                saved={result['saved']}
+            ''')
+            if hasattr(result['instance'], 'project'):
+                msg += f"\nproject={result['instance'].project}"
+        msg += '\n'
+    return msg
+
+
+def replace_attribute_on_elements(source, target, model_helpers_with_attributes, save):
```

Comment:
Also not needed, the containing code should be just in `handle`.

Reply:
yes, moved most of the code to `handle`